lib/uart.cpp

```diff
@@ -56,16 +61,21 @@ using namespace irve::internal;
  * --------------------------------------------------------------------------------------------- */
 
 Uart::Uart() {
-    //TODO
+    this->regs = {static_cast<uint8_t>(irve_fuzzish_rand()), static_cast<uint8_t>(irve_fuzzish_rand()), static_cast<uint8_t>(irve_fuzzish_rand()), static_cast<uint8_t>(irve_fuzzish_rand()), static_cast<uint8_t>(irve_fuzzish_rand()), static_cast<uint8_t>(irve_fuzzish_rand()), static_cast<uint8_t>(irve_fuzzish_rand())};
```

Comment:
Now that the regs are grouped in a struct, you could do this instead:

    irve_fuzzish_meminit(&this->regs, sizeof(this->regs));

Reply:
Fair enough. I personally think the current way is more clear but if you think `irve_fuzzish_meminit(&this->regs, sizeof(this->regs));` is better, we can change this.

Reply:
Yeah I see where you're coming from, I'm good with keeping it as is